package.json

```diff
@@ -67,7 +67,7 @@
     "vite": "^5.4.4"
   },
   "dependencies": {
-    "bits-ui": "^0.22.0",
+    "bits-ui": "^2.8.12",
```

Comment:
This upgrade remove `Value` from the bits-ui exports for Select.

So we get an error when trying to load the storybook for http://localhost:6006/?path=/story/ui-components-atoms-select--default-select